spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/ContractDownloader.java

```diff
@@ -193,8 +193,8 @@ private String surroundWithSeparator(String string) {
 	}
 
 	private String wrapWithAntPattern(String path) {
-		String changedPath = path.replace(File.separator, "/");
-		return "**" + surroundWithSeparator(changedPath).replace(File.separator, "/")
+		String changedPath = surroundWithSeparator(path).replace(File.separator, "/");
```

Comment:
The test wants:
**/a/b/c/d/**
what is generated before is **//a/b/c/d//**
This corrects that by adjusting the order when createding the AntPattern